### PR TITLE
First draft of newsletter index

### PR DESCRIPTION
**Pull request by bethaitman**
I've checked out the formatting in http://livesphinx.herokuapp.com/, so hopefully should be okay. 

Might be worth looking at a bigger overhaul of the content/toc, but this is a first step.

### CODE REVIEW

**Comment by plaindocs** [comment]
Woah, this is EXCITING! :clap: :clap: :clap: :clap: @bethaitman

File: docs/newsletter.rst [hunk 1]
```diff
@@ -42,3 +42,222 @@ If you have any questions or want to contribute to the newsletter, go ahead and
...
+^^^^^^^^^^^^
+
+- `Enforcing UI style guides </blog/newsletter-june-2017/#the-enforcer-ui-style-guides-edition>`_
+- `Style guides we like </blog/newsletter-november-2016/#a-quick-guide-to-style-guides>`_
+- `One style guide for all types of writing? </blog/newsletter-september-2016/#one-style-guide-or-two>`_
+
+Encouraging contributions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+See also: helping others to write
```
Comment (plaindocs) [inline docs/newsletter.rst:88]:
```suggestion
See also `helping others to write <#helping-others-to-write>`_ 
```

Comment (plaindocs) [reply docs/newsletter.rst:88]: ~Not 100% sure about this syntax for an in page anchor~

Comment (ericholscher) [reply docs/newsletter.rst:88]: This is worth doing, but not going to block merge on it 👍

**Review by ericholscher — APPROVED** [review]
Looks great with the heading update. 💯

**Pull request merged**